### PR TITLE
expandable table

### PR DESCRIPTION
**Pull request by qbalukom**

### CODE REVIEW

File: core-ui/src/shared/components/GenericList/GenericList.scss [hunk 1]
```diff
@@ -36,6 +36,7 @@
     }
   }
 
+  /*
```
Comment (akucharska) [inline core-ui/src/shared/components/GenericList/GenericList.scss]: If this is not necessary then please remove it

**Review by akucharska — COMMENTED** [review]
It would be good to address the issue with the `margin` at the `columns` component. If not now then create an issue for it.
Screen Shot 2022-07-05 at 11 07 01
Screen Shot 2022-07-05 at 11 06 48

File: core-ui/src/shared/components/GenericList/GenericList.scss (hunk 1 above)
Comment (qbalukom) [reply core-ui/src/shared/components/GenericList/GenericList.scss]: already did :)

**Review by akucharska — DISMISSED** [review]

**Review by akucharska — APPROVED** [review]

**Review by NHingerl — APPROVED** [review]

**Pull request merged**